src/v2/zcl_oapi_generator_v2.clas.abap

```diff
@@ -328,4 +342,15 @@ CLASS zcl_oapi_generator_v2 IMPLEMENTATION.
       ENDLOOP.
     ENDLOOP.
   ENDMETHOD.
+
+
+  METHOD shorten_content_type_name.
+    rv_short_name =
+      SWITCH #( iv_content_type
```

Comment:
this SWITCH must be downported to 702 syntax

Reply:
I have removed this method. It was replaced with a class that handles the majority of the media types instead.